tests/Immediate.Validations.FunctionalTests/IntegrationTests/LocalizationTests.cs

```diff
@@ -0,0 +1,60 @@
+using System.Globalization;
+using Immediate.Validations.Shared;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace Immediate.Validations.FunctionalTests.IntegrationTests;
+
+public sealed partial class LocalizationTests
+{
+	[Validate]
+	public sealed partial record ValidateRecord : IValidationTarget<ValidateRecord>
+	{
+		[GreaterThan(0)]
+		public required int Id { get; init; }
+	}
+
+	[Fact]
+	public void DefaultLocalizedMessage()
```

Comment:
I'd like to see tests based on the `.Shared` localizer as well.

Reply:
Before I write tests for this... I'm facing a race condition when running tests that I can't seem to find the source of... sometimes the tests fail as the error message is "SomeAttribute" instead of the error message. I replaced the strings in the localizer and put exceptions where I thought this could be happening, and nothing. I think this is manifesting itself through the handling of the DefaultMessage.

Reply:
Added tests.